### PR TITLE
updated how to create a section in V8

### PR DESCRIPTION
**Pull request by skartknet**

### CODE REVIEW

**Comment by sofietoft** [comment]
Hi @skartknet 

This is good stuff! 💪
Would you mind taking a crack at splitting the article up in two? One for Umbraco 7 and one for Umbraco 8?
We're using YAML tags and file names for the versioning. You can read more about the details in the Contributing section.

I'd also be happy to do it as well. Just let me know 😁
/Sofie

**Comment by skartknet** [comment]
no problem, I have created a new commit with the changes and also improved it with some fixes and new images. :)

File: Extending/Section-Trees/sections-v8.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+![Add Section for User](images/add-custom-section-v8.png)
+
+### Adding a Language Translation (get rid of the square brackets)
+
+![Custom Section appears displaying Alias](images/custom-section-alias-v8.png)
+
+When your new custom section appears, you'll notice only the section 'Alias' is displayed inside square brackets. This is because Umbraco caters for Multiple Languages in the backoffice, and is looking for a translation file for the current backoffice culture, containing a translation key for your custom section alias.
+
+Create a /lang folder in the folder where you are creating the implementation for your custom section(if not create one in the app*plugins folder eg */app_plugins/favouritethings/lang\*)
```
Comment (sofietoft) [inline Extending/Section-Trees/sections-v8.md]:
```suggestion
Create a /lang folder in the folder where you are creating the implementation for your custom section. If not create one in the App_Plugins folder eg. */App_Plugins/MyFavouriteThings/lang/*.
```

File: Extending/Section-Trees/sections-v8.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+inside this folder create a file called **en-us.xml** this is the 'default' fallback language translation file, and add the following definition:
```
Comment (sofietoft) [inline Extending/Section-Trees/sections-v8.md]:
```suggestion
Inside this folder create a file called **en-us.xml**. This is the 'default' fallback language translation file. Add the following definition:
```

**Comment by sofietoft** [comment]
Awesome! 🎉 Thanks @skartknet 

Just a few grammar corrections (sorry, for being such a grammar nerd 🙈 ), and we're good to go!

**Comment by sofietoft** [comment]
Looks great @skartknet 😄

**Pull request merged**